recipe/meta.yaml

```diff
@@ -1,5 +1,7 @@
 {% set name = "simpeg-drivers" %}
 {% set version = "0.1.1" %}
+{% set python_min = 3.10 %}
+{% set python_max = 3.11 %}
```

Comment:
@SophieCurinier, note for next version: max will have to be changed to 3.12, as it will support Python 3.11

the upcoming versions of our other packages will support up to Python 3.12
But, packages dependent on simpeg do not support 3.12 yet (only 3.10, 3.11)